File: src/main/java/com/vokerg/blog/VokergBlog/model/Comment.java
package com.vokerg.blog.VokergBlog.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection="comments")
@Data
public class Comment {
    @Id
    private String id;
    private String idAuthor;
    private String idArticle;
    private String idParentComment;
    private String authorName;
    private String text;
    private String title;
}

File: client/src/app/model/comment.ts
import {Article} from "./article";

export class Comment {
  id: string;
  idAuthor: string;
  idArticle: string;
  authorName: string;
  title: string;
  text: string;
  article: Article;
  liked: boolean;
  likeCount: number;
  idParentComment: string;
  constructor() {
    this.id=null;
    this.idAuthor="";
    this.idArticle="";
    this.authorName="";
    this.title="";
    this.text="";
    this.liked = false;
    this.likeCount = 0;
    this.idParentComment="";
  }
}
